docs/source/app-dev/grpc/error-codes.rst

```diff
@@ -8,11 +8,11 @@
 :tocdepth: 2
 
 
-Error Codes
-###########
+Understanding Error Codes
+#########################
```

Comment:
```suggestion
Error Codes
###########
```

In a sense, all documentation pages are about understanding something. Are you sure we need this in this title? The title seems pretty simple and self-explanatory as it were.